033-OSSDatabaseMigration/Coach/04-offline-cutover-validation.md

```diff
@@ -0,0 +1,8 @@
+# Challenge 4: Offline Cutover and Validation
+
+[< Previous Challenge](./03-offline-migration.md) - **[Home](./README.md)** - [Next Challenge >](./05-online-migration.md)
+
+## Coach Tips
+
+1) It's a good idea to have the attendee register themselves as a new user so they can then go into psql/mysql/Azure Data Studio/MySQL WorkBench and check the users table to see that the data has indeed changed. Keep in mind that registering users will not work unless Google reCAPTCHA has been reconfigured. This is mentioned in the [Prerequisites](./00-prereqs.md) 
```

Comment:
wrap the psql/mysql/.. path in backticks so that it is monospaced and look separate from the rest of the text.

Reply:
Fixed with commas instead since it was intended to reference various DB tools